subsys/testsuite/ztest/Kconfig

```diff
@@ -87,6 +87,13 @@ config ZTEST_NO_YIELD
 	  yielding to the idle thread may put the board into a low power state
 	  where a debugger cannot connect to it.
 
+config ZTEST_WIP
+	bool "Indicates the test is a work in progress."
+	default y
+	help
+	  Used in conjunction with Z_TEST_SKIP_IFDEF, you can skip tests that
+	  are still in development.
+
```

Comment:
I don't think having a single config is useful. I would expect every application to define _N_ of these 1:1 with each feature being developed/debugged.